src/test/java/org/jabref/logic/bibtex/BibEntryAssert.java

```diff
@@ -35,9 +35,9 @@
      */
     public static void assertEquals(Class<?> clazz, String resourceName, BibEntry entry)
             throws IOException {
-        Assert.assertNotNull(clazz);
-        Assert.assertNotNull(resourceName);
-        Assert.assertNotNull(entry);
+        Assertions.assertNotNull(clazz);
```

Comment:
I would suggest, using a static import as before makes it more readable

Reply:
Done.